tests/test_native_serve.py

```
@@ -273,38 +483,557 @@ async def run():
+            victim_result["status"] = r.status_code
+            victim_result["body"] = r.text[:200]
+        except Exception as e:
+            victim_result["status"] = -1
+            victim_result["body"] = str(e)
+
+    t_anchor = threading.Thread(target=post_anchor)
+    t_anchor.start()
+    # Wait until the anchor is in flight (busy → a transcribe is running),
+    # then queue the victim behind it.
+    if not _wait_busy_via_health(base_url, 30.0):
+        tr.check("cancel test: anchor in flight", False, "server never busy")
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)